core/src/test/java/nl/nn/adapterframework/jdbc/dbms/TestBlobs.java

```diff
@@ -11,23 +11,48 @@
 import java.io.Reader;
 import java.io.Writer;
 import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
+import java.sql.SQLException;
 import java.util.function.Consumer;
 
 import org.apache.commons.io.input.ReaderInputStream;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
 
+import nl.nn.adapterframework.dbms.JdbcException;
 import nl.nn.adapterframework.jdbc.JdbcQuerySenderBase.QueryType;
-import nl.nn.adapterframework.jdbc.JdbcTestBase;
+import nl.nn.adapterframework.testutil.TransactionManagerType;
+import nl.nn.adapterframework.testutil.junit.DatabaseTest;
+import nl.nn.adapterframework.testutil.junit.DatabaseTestEnvironment;
+import nl.nn.adapterframework.testutil.junit.WithLiquibase;
+import nl.nn.adapterframework.util.JdbcUtil;
 import nl.nn.adapterframework.util.StreamUtil;
 
-@Ignore("Tests for Blobs take too much time and memory to test regularly")
-public class TestBlobs extends JdbcTestBase {
+@Disabled("Tests for Blobs take too much time and memory to test regularly")
```

Comment:
Since these tests have been disabled, did you test the old tests still worked? And that the new tests also work?

Reply:
The @Disabled is a replacement for @Ignore in Junit 5

Reply:
Yes, Tim knows that. That's why he asks his questions 😉

Reply:
ah makes sense, I misunderstood the comment

Reply:
De tests werken nog.
Het resultaat is 12/30 passed test, 18/30 ignored tests

Reply:
En als je in deze test-class ook even het vlaggetje `testBigBlobs` op `true` zet, draaien alle tests dan?

Reply:
Dan werken ze ook nog inderdaad